core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -2615,7 +2615,7 @@ class ReplicaManagerTest {
 
   @Test
   def testStopReplicaWithDeletePartitionAndExistingPartitionAndNewerLeaderEpochAndIOException(): Unit = {
-    testStopReplicaWithExistingPartition(2, true, true, Errors.KAFKA_STORAGE_ERROR)
+    testStopReplicaWithExistingPartition(2, true, true, Errors.NONE)
```

Comment:
Could we add a comment on when this doesn't hit the storage error?

Reply:
I've added some comments about this change.